packages/common/src/compiler/exploreCompiler.ts

```diff
@@ -241,8 +243,32 @@ const compileMetricReference = (
     };
 };
 
-export const renderSqlType = (sql: string, type: MetricType): string => {
+const renderSqlType = (
+    sql: string,
+    type: MetricType,
+    targetDatabase: SupportedDbtAdapter,
+    percentile: number | undefined = 50,
```

Comment:
what is the case of this arg fallbacking to 50?

Reply:
It can happen if they don't define `percentile` in the yml or when they create a custom metric via the UI. 
Found easier to have a default value than to throw an error.